HLint.hs

```diff
@@ -16,3 +16,4 @@ ignore "Use const"
 ignore "Use fmap"
 ignore "Use list literal"
 ignore "Use record patterns"
+ignore "Use =<<"
```

Comment:
Perhaps we can contain that to one module?  I have no strong opinion here, but it would be nice to keep reminded by hlint if we bump into this again, and decide on a case-by-case basis.